extensions/wikia/PortableInfobox/services/Parser/XmlParser.php

```diff
@@ -51,8 +51,17 @@ public function getDataFromNodes( \SimpleXMLElement $xmlIterable ) {
 	 */
 	public function getDataFromXmlString( $xmlString ) {
 		wfProfileIn( __METHOD__ );
+		libxml_use_internal_errors( true );
 		$xml = simplexml_load_string( $xmlString );
 		if ( $xml === false ) {
+			$errors = libxml_get_errors();
+			foreach ( $errors as $xmlerror ) {
+				\Wikia\Logger\WikiaLogger::instance()->debug( "PortableInfobox XML Parser problem", [
```

Comment:
Per our offline discussion, might be worth logging the libXMLError accordingly through the logger ( warning / error / critical )